src/services/groups.service.ts

```diff
@@ -0,0 +1,96 @@
+import { PrismaClient, Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import { HttpException } from '@exceptions/HttpException';
+import { isEmpty } from '@utils/util';
+
+class GroupService {
+  public group = new PrismaClient().group;
+
+  public async findAllGroup(): Promise<Group[]> {
+    const allGroup: Group[] = await this.group.findMany();
+    return allGroup;
+  }
+
+  public async findGroupById(groupId: number): Promise<Group> {
+    if (isEmpty(groupId)) throw new HttpException(400, "groupId is empty");
+
+    const findGroup = await this.group.findUnique({ where: { id: groupId }});
+    if (!findGroup) throw new HttpException(409, "Group doesn't exist");
+
+    return findGroup;
+  }
+
+  public async createGroup(GroupData: CreateGroupDto): Promise<Group> {
+    if (isEmpty(GroupData)) throw new HttpException(400, "GroupData is empty");
+
+    const createGroupData: Group = await this.group.create({ data: { ...GroupData, }});
+    return createGroupData;
+  }
```

Comment:
Change the name of the function. As we are on the **Groups Service**, if we have a method that create a **group** we can simply call it like **create** because it tells that we are creating the entity from which our service is responsible.

Add braces and break lines like mentioned on **findGroupById** method.

On line 23 change **GroupData** to **groupData** to keep camelCase variables.
On line 24 and 26 try to avoid Magic Numbers.
On line 26 the variable name could be **groupCreated**, also its not needed to infer the Group type since Prisma have a great integration with typescript. And you can do `data: groupData` instead of opening a object and spreading it.